app/subscriber/src/features/router/AppRouter.tsx

```diff
@@ -31,6 +31,7 @@ export const AppRouter: React.FC = () => {
   const navigate = useNavigate();
 
   React.useEffect(() => {
+    window.snowplow('trackPageView');
```

Comment:
I am doubtful this will work as it will fire every time the navigate function is changed.  But we can give it a try.

Reply:
Isn't the navigate function going to change only when the page view changes? When else will it be fired?

Reply:
It 'might' change, but you would have to test it.  It's a function, so unless it's maintaining state it should rarely change.